lib/bore.lua

```diff
@@ -360,14 +390,15 @@ m.cleanInventory = function ()
   for slot = 1, 16, 1 do
     if turtle.getItemCount(slot) > 0
       and slot ~= bucketSlot
-      and not wants(turtle.getItemDetail(slot, true)) then
+      and not wants.wants(turtle.getItemDetail(slot, true)) then
         turtle.select(slot)
         turtle.drop()
     end
   end
 end
 
-m.cleave = function(dimensionVector)
+m.cleave = function(dimensionVector, options)
+  wants.setProfile(options.wants)
```

Comment:
This seems like the only usage of options. Because it is so closely tied to the command line options, it might be worth just lifting out into bin\cleave.lua, similarly to how `setChest` is called. Especially since this call will no longer be mandatory if the default is selected by default.